.travis.yml

```diff
@@ -2,10 +2,10 @@ language: ruby
 cache: bundler
 
 rvm:
+  - 2.4.0
   - 2.3.0
   - 2.2.0
   - 2.1.0
-  - 2.0.0
```

Comment:
I think that we can do it, because [Support of Ruby 2.1 has ended](https://www.ruby-lang.org/en/news/2017/04/01/support-of-ruby-2-1-has-ended/)